src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -79,11 +79,11 @@ export interface ReleaseNote {
 
 // eslint-disable-next-line no-shadow
 export enum ReleaseType {
-  AdHoc = 'Ad Hoc',
-  Experimental = 'Experimental',
+  AdHoc = 'Ad Hoc Statistics',
```

Comment:
I would rename `AdHoc` & `Experimental` to conform with the naming convention here. i.e. `AdHocStatistics` & `ExperimentalStatistics`